Registry/Registry.EM_COMMON

```diff
@@ -807,13 +808,18 @@ state    real   DZR             l        em      -         Z     r        "DZR"
 state    real   DZB             l        em      -         Z     r        "DZB"            "THICKNESSES OF WALL LAYERS"                      "m"
 state    real   DZG             l        em      -         Z     r        "DZG"            "THICKNESSES OF ROAD LAYERS"                      "m"
 state    real   URB_PARAM      i{urb}j  misc      1        -     i1       "URB_PARAM" "NUDAPT_NBSD Urban Parameters" "parameter"
-state    real   LP_URB2D       ij       misc      1        -     ir       "BUILD_AREA_FRACTION" "BUILDING PLAN AREA DENSITY"   "dimensionless"
+state    real   LP_URB2D       ij       misc      1        -     i01r     "BUILD_AREA_FRACTION" "BUILDING PLAN AREA DENSITY"   "dimensionless"
 state    real   HI_URB2D       i{uhi}j  misc      1        Z     ir       "HEIGHT_HISTOGRAMS" "DISTRIBUTION OF BUILDING HEIGHTS" "dimensionless"
 state    real   LB_URB2D       ij       misc      1        -     ir       "BUILD_SURF_RATIO" "BUILDING SURFACE AREA TO PLAN AREA RATIO" "dimensionless"
 state    real   HGT_URB2D      ij       misc      1        -     ir       "BUILD_HEIGHT" "AVERAGE BUILDING HEIGHT WEIGHTED BY BUILDING PLAN AREA" "m"
-state    real   MH_URB2D       ij       misc      1        -     ir       "MH_URB2D"  "Mean Building Height" "m"
+state    real   MH_URB2D       ij       misc      1        -     i01r     "MH_URB2D"  "Mean Building Height" "m"
 state    real   STDH_URB2D     ij       misc      1        -     ir       "STDH_URB2D" "Standard Deviation of Building Height" "m2"
 state    real   LF_URB2D       i{udr}j  misc      1        Z     ir       "LF_URB2D"   "Frontal Area Index" "dimensionless"
+state    real   ZD_URB2D       ij       misc      1        -     i1       "ZD_URB2D"   "Zero-plane Displacement" "m"
+state    real   Z0_URB2D       ij       misc      1        -     i01r     "Z0_URB2D"   "Roughness length for momentum"  "m"
+state    real   LF_URB2D_S     ij       misc      1        -     i01r     "LF_URB2D_S" "Frontal area index (no wind directional dependency)" ""
+# AHE with month and hour dimension flattened to one dimension, Jan = (0:23), Feb = (24:47)
+state    real   AHE            i{m_hr}j  misc       1        -     i01r   "AHE"        "Anthropogenic heat emission" "W m-2"
```

Comment:
@cenlinhe You should consider packaging these new arrays to reduce memory if the option is not used. Once the variable is part of the registry, I don't think you can use optional variable any more. They are in the code, present. If people don't have it in the metgrid input files, and the package sets the arrays to a dimension of 1, and the code should have conditionals to prevent accessing/using these arrays.

Reply:
Agreed. Looks safe to package it and it is used with options 1 and 2 of this namelist variable.